<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>오늘의 날씨</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>지역 입력 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_area">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="weather_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>날씨 조회</string>
    </property>
   </widget>
   <widget class="QLabel" name="area_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 102, 255);</string>
    </property>
    <property name="text">
     <string>지역 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weather_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>120</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);</string>
    </property>
    <property name="text">
     <string>날씨</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temper_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>120</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>온도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="yesterday_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>어제와의 날씨 비교</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 115, 0);</string>
    </property>
    <property name="text">
     <string>체감 온도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 193, 193);</string>
    </property>
    <property name="text">
     <string>미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 115);</string>
    </property>
    <property name="text">
     <string>초미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="senseTemper_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width:2px;
border-color: rgb(255, 115, 0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dust1_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width:2px;
border-color:  rgb(193, 193, 193);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dust2_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width:2px;
border-color: rgb(232, 232, 115);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
